dev-libs/spdlog/spdlog-1.5.0.ebuild

```diff
@@ -21,19 +21,29 @@ SLOT="0/1"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
+BDEPEND="
+	virtual/pkgconfig
+"
 DEPEND="
-	>=dev-libs/libfmt-5.0.0
+	>=dev-libs/libfmt-5.3.0:=
 "
 RDEPEND="${DEPEND}"
 
-PATCHES=( "${FILESDIR}/${PN}-1.0.0-unbundle-fmt.patch" )
+#PATCHES=(
+#	"${FILESDIR}/${PN}-1.4.2-fix-compilation.patch"
+#)
 
-src_configure() {
-	rm -r include/spdlog/fmt/bundled || die
+src_prepare() {
+	cmake_src_prepare
+	rm -r include/spdlog/fmt/bundled || die "Failed to delete bundled libfmt"
+}
 
+src_configure() {
 	local mycmakeargs=(
-		-DSPDLOG_BUILD_EXAMPLES=no
 		-DSPDLOG_BUILD_BENCH=no
+		-DSPDLOG_BUILD_EXAMPLE=no
```

Comment:
Should this be `EXAMPLES` or `EXAMPLE`?

I personally don't know, but I want to make sure this change is correct and not a mistake.

Reply:
yes, it's correct. From CMakeLists.txt `option(SPDLOG_BUILD_EXAMPLE "Build example" ${SPDLOG_MASTER_PROJECT})`